www/default.js

```diff
@@ -143,9 +143,11 @@ function change_cost(duration) {
     per_time = elem.data("pricing")[g_settings.region];
     if (per_time && !isNaN(per_time)) {
       per_time = (per_time * multiplier).toFixed(3);
-      elem.text("$" + per_time + " " + duration);
+      elem.empty();
+      elem.append('<span sort="' + per_time + '">$' + per_time + ' ' + duration + '</span>');
```

Comment:
Adding span through elem.text shows the tags in the datatable too: [
![span](https://user-images.githubusercontent.com/12442986/26892343-94fb30b6-4bea-11e7-85a1-81778c91e3e6.png)
](url)